.tool-versions

```diff
@@ -0,0 +1,2 @@
+nodejs 14.19.3
+python 3.11.1
```

Comment:
very nice! for the people already on nvm (most of the team) it might be good to add an `.nvmrc` also. Not sure if there is an equivalent with pyenv

Reply:
I think there is a pyenv. I'll check it out

Reply:
Done